homeassistant/components/rfxtrx/__init__.py

```diff
@@ -409,6 +409,25 @@ def assumed_state(self):
         """Return true if unable to access real state of entity."""
         return True
 
+    @property
+    def unique_id(self):
+        """Return unique identifier of remote device."""
+        return f"{slugify(self._event.device.type_string.lower())}_{slugify(self._event.device.id_string.lower())}"
```

Comment:
`slugify` is expensive. We should avoid calling it repeatedly, like on every state update.